index.html

```diff
@@ -0,0 +1,63 @@
+<!-- Milestone 1, John Ieng -->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="../styles.css" />
+        <title>Yums</title>
+    </head>
+    <body>
+        <!-- page header -->
+        <header>
+            <nav class="navbar">
+                <div class="navbar-logo navbar-item">
+                    <a class="nav-link" href="../index.html">Yums</a>
+                </div>
+                <ul class="navbar-items">
+                    <li class="navbar-item">
+                        <a class="nav-link" href="../index.html">Home</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a class="nav-link" href="../about.html ">About Me</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <!-- page content -->
+        <main>
+            <h1 class="title">Check out some of our favorites!</h2>
+                <!-- list of recipes -->
+                <div class="container-recipes">
+                    <div class="card">
+                        <h2 class="card-title">Lemon Bars</h2>
+                        <div class="card-body">
+                            <img class="card-img" src="https://preppykitchen.com/wp-content/uploads/2020/02/lemon-bars-blog-2a-1140x1271.jpg" width="250" height="250">
+                            <p class="card-disc">You'll love these bright and zingy lemon bars with a crisp
+                                shortbread crust and luscious lemon filling.</p>
+                            <a class="card-btn" href="/recipes/recipe-1.html">Go</a>
```

Comment:
Similar issue here - you want to specify the current directory with `./recipes/...` instead of `/recipes/...`